roles/vpn/templates/etc_openvpn_server.conf.j2

```diff
@@ -190,6 +190,7 @@ ifconfig-pool-persist ipp.txt
 ;push "redirect-gateway def1 bypass-dhcp"
 push "redirect-gateway def1"
 push "dhcp-option DNS 10.8.0.1"
+push "dhcp-option DNS 8.8.8.8"
```

Comment:
I think hard coding Google's DNS servers into sovereign might run modestly contrary to the project's spirit. Perhaps the line could be removed?

Separating out dnsmasq might make sense. Is it required for anything besides openvpn?